Client/Clique/Modules/EventDetail/EventDetailViewController.swift

```diff
@@ -20,32 +22,172 @@ class EventDetailViewController: UIViewController {
     @IBOutlet private var _eventParticipantsNum: UILabel!
     @IBOutlet private var _eventCost: UILabel!
     @IBOutlet private var _eventCategory: UILabel!
+    @IBOutlet private var buttonJoinEvent: UIButton!
+    @IBOutlet private var cosmosView: CosmosView!
+    
+    
+    private var eventServices = EventServices()
+    private var status = 0
+    private var alertMessage = ""
+    private var eventStatus = Event.status.done
```

Comment:
This value should be calculated and used from `Event` object itself, the `ViewController` should not know what is the status of event.